Source/JavaScriptCore/jit/JITPropertyAccess.cpp

```diff
@@ -1890,12 +1960,40 @@ MacroAssemblerCodeRef<JITThunkPtrTag> JIT::slow_op_get_from_scopeGenerator(VM& v
     Jump jumpToHandler = jit.jump();
 
     LinkBuffer patchBuffer(jit, GLOBAL_THUNK_ID, LinkBuffer::Profile::ExtraCTIThunk);
-    patchBuffer.link(operation, FunctionPtr<OperationPtrTag>(operationGetFromScope));
+    patchBuffer.link(operation, FunctionPtr<OperationPtrTag>(operationFunction));
     auto handler = vm.getCTIStub(popThunkStackPreservesAndHandleExceptionGenerator);
     patchBuffer.link(jumpToHandler, CodeLocationLabel(handler.retaggedCode<NoPtrTag>()));
     return FINALIZE_THUNK(patchBuffer, JITThunkPtrTag, "Baseline: slow_op_get_from_scope");
 }
 
+void JIT::emit_op_resolve_and_get_from_scope(const JSInstruction* currentInstruction)
+{
+    emit_op_resolve_scope_helper<OpResolveAndGetFromScope>(currentInstruction, currentInstruction->as<OpResolveAndGetFromScope>().m_dst);
+    emit_op_get_from_scope_helper<OpResolveAndGetFromScope>(currentInstruction, currentInstruction->as<OpResolveAndGetFromScope>().m_dst);
```

Comment:
We should refactor this in such a way we don't need to write out the intermediate resolve_scope result to the stack. There is no need to do that, so we should avoid it

Reply:
The current implementation stores the intermediate resolve_scope result to the `m_dst` of op_resovle_and_get_from_scope. So, it doesn't need extra space in the stack to keep it. Are you proposing that we should store it to an GPR instead? Could that perform better performance in read and write?